app/code/Magento/Sales/Block/Adminhtml/Order/Address/Form.php

```diff
@@ -136,4 +136,12 @@ public function getFormValues()
     {
         return $this->_getAddress()->getData();
     }
+
+    /**
+     * @inheritDoc
+     */
+    protected function getAddressStoreId()
```

Comment:
Unfortunately, we are not allowed to add new public and protected methods to the classes according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
@sidolov what is your suggested solution in this case?

Reply:
The best way to resolve such issue - to extract code used in both classes to the separate class.

Reply:
This is not an API class